util/minetestmapper.py

```diff
@@ -131,6 +132,8 @@ def readS32(f):
   --drawscale
   --drawplayers
   --draworigin
+  --region <xmin>:<xmax>,<zmin>:<zmax>
+  --geometry <xmin>:<zmin>+<width>+<height>
   --drawunderground
 Color format: '#000000'"""
```

Comment:
Would be nice to have something here talk about the colors.txt file and where this script tries to find it.